apps/server/src/modules/files-storage/service/files-storage.service.ts

```diff
@@ -36,14 +31,44 @@ import {
 import { IGetFileResponse } from '../interface';
 import { CopyFileResponseBuilder, FileRecordMapper, FilesStorageMapper } from '../mapper';
 import { FileRecordRepo } from '../repo';
+import { FileRecord, FileRecordPropertiesFactory, IFileRecordParams } from '../domain';
+import { FileSecurityCheck } from '../repo/filerecord.entity';
 
+// TODO: Ticket for rename EntityId to more generic Name
+
+export interface IFilesStorageRepo {
```

Comment:
Warum ist das Interface vom Repo im Service definiert und nicht in einer separaten Datei?